app/views/licence/_authority_details.html.erb

```diff
@@ -10,9 +10,9 @@
           <li class="active">Overview</li>
         <% end %>
         <% @licence_details.authority['actions'].keys.uniq.each do |action| %>
-          <% if @licence_details.action == action %>
+          <% if @licence_details.action == action && @licence_details.uses_licensify(action) %>
             <li class="active"><%= "How to #{action}" %></li>
-          <% else %>
+          <% elsif @licence_details.uses_licensify(action) %>
```

Comment:
Are we sure that the right thing to do is not render any navigational links to the "how to `whatever`" action/link if it doesn't use licensify?  If I'm reading the code correctly, we can still render the navigational link, but instead of showing a big green "do it on licensing" button we'd instead render the "licensing unavailable" page which suggests they contact their council directly.

I don't know if that's correct or not based on the contents of the api response and what it actually means though.

This is sort of the same as my comment where an action has multiple links - is it better to show nothing if they don't use licensify, or show something and point people onwards to the council for more help?

Reply:
Removed, this is handled in the `_action.html.erb` instead.